eng/restore-toolset.ps1

```diff
@@ -23,7 +23,7 @@ function InitializeCustomSDKToolset {
   InstallDotNetSharedFramework "5.0.0"
   InstallDotNetSharedFramework "6.0.0-rc.2.21452.2"
```

Comment:
Cant this be changed to just ``6.0.0``?

Reply:
unknown, I don't know the implications of that change. IMO it's beyond the scope of this PR.